src/splitInteger.test.js

```diff
@@ -4,18 +4,22 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
+  expect(splitInteger(8, 2))
+    .toEqual([4, 4]);
 });
 
 test(`should return a part equals to a value
   when splitting into 1 part`, () => {
-
+  expect(splitInteger(5, 1))
+    .toEqual([5]);
 });
 
 test('should sort parts ascending if they are not equal', () => {
-
+  expect(splitInteger(21, 3))
+    .toEqual([7, 7, 7]);
```

Comment:
'should sort parts ascending if they are not equal', so one part value in the array should be less when anothers

Reply:
Fixed